Adding comment support

collinpburns opened the pull request:
Hey @slobatch , I added support for issue commented, issue comment updated, issue comment deleted, and issue updated events.

Thanks for putting this script together!

slobatch commented on src/jira_to_chime/jira_to_chime.py:
```diff
@@ -1,106 +1,117 @@
+def lambda_handler(event, context):
+
+    try:
+        post_to_chime(chime_webhook_url = hook, message = assemble_message(event))
```
We should probably keep this as `os.environ['CHIME_HOOK']` so that users can easily modify their chime webhook URL in the lambda environment functions.

(Where are you defining `hook`?)

collinpburns replied:
Sorry this was from another test I was doing with multiple webhooks.

slobatch commented on src/jira_to_chime/jira_to_chime.py:
```diff
@@ -1,106 +1,117 @@
+    jira_base_url = os.environ['JIRA_URL']
+
+    #print (jira_event_type)
+    #print (jira_webhook_event)
```
Let's remove these comments to keep the script clean. :)

slobatch commented:
@collinpburns 

Looks like you changed the whitespace with your commit. Do you mind addressing that so that we can keep the git-blame clean?

Might you also be able add a couple of test payloads with comment changes (anonymized, obviously) to facilitate testing locally before I merge?

collinpburns committed:
Add files via upload

collinpburns committed:
Add files via upload

collinpburns committed:
Delete jira_to_chime.py

collinpburns committed:
Add files via upload

collinpburns commented:
Hey @slobatch 

I've updated the whitespace and added in test payloads as well as addressing the other concerns. Please take a look and let me know if I missed anything else. 

Thanks,
Collin

slobatch commented on test/example_comment_created_body.json:
```diff
@@ -0,0 +1,42 @@
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+{
```
@collinpburns -- tried testing using this payload today. is this really what you get from JIRA? the `body` doesn't contain `issue_event_type_name`, which `jira_to_chime.py` expects on line 43. 

were you able to test it with this payload?

slobatch reviewed (changes requested):
Test payload for comment_created doesn't seem to work. Is the problem with the test payload or with the code?

collinpburns committed:
Add files via upload

Recreated test content and verified all working locally.

collinpburns commented:
Hi @slobatch 

I recreated all three test content files and verified that they are working locally from my computer. 

Thanks,
Collin

slobatch commented:
@collinpburns the updated payloads seem to work well for me. :)

One last request -- do you mind squashing your commits? https://blog.carbonfive.com/2017/08/28/always-squash-and-rebase-your-git-commits/

slobatch commented:
Merging with the Github's convenient squash-and-merge feature.
